src/bundler.js

```diff
@@ -247,30 +247,36 @@ function processJsBundle(options, jsBundle, bundleDir, jsFiles, bundleName, cb)
 
     var allJsArr = [], allMinJsArr = [], index = 0, pending = 0;
     var whenDone = function () {
-        var allJs = concat.files({
+
+        concat.files({

```

Comment:
Since I made `concat` return a promise, I also converted this to a promise chain rather than a bunch of nested callbacks. It's probably easier to just read this code in it's final form rather than trying to make sense of the diff. I think it's easier to read and understand now, though.